### PR TITLE
changed Ghana Polytechnics to Technical University

### PR DESCRIPTION
**Pull request by co-bby**
I live in Ghana and Ghana has changed some of  its polytechnnics to Technical universities. If you want to test the validity you can try loging  onto the changed domains. You will see it is not working no more.

### CODE REVIEW

File: world_universities_and_domains.json [hunk 1]
```diff
@@ -44915,13 +44915,13 @@
...
-      "http://www.accrapolytechnic.edu.gh/"
+      "https://atu.edu.gh/"
     ],
-    "name": "Accra Polytechnic",
+    "name": "Accra Technical University",
     "alpha_two_code": "GH",
     "state-province": null,
     "domains": [
-      "accrapolytechnic.edu.gh"
+      "https://atu.edu.gh/"
```
Comment (yigitguler) [inline world_universities_and_domains.json:44924]: Domains should not include the https:// part. Also the trailing slash should be removed from the domain entries.

**Review by co-bby — COMMENTED** [review]
I have fixed the domain problem

File: world_universities_and_domains.json [hunk 2]
```diff
@@ -45059,25 +45059,25 @@
...
-      "http://www.kpoly.edu.gh/"
+      "https://www.kstu.edu.gh/"
     ],
-    "name": "Kumasi Polytechnic",
+    "name": "Kumasi Technical University",
     "alpha_two_code": "GH",
     "state-province": null,
     "domains": [
-      "kpoly.edu.gh"
+      "www.kstu.edu.gh"
```
Comment (yigitguler) [inline world_universities_and_domains.json:45080]: As far as I see, on this university, email domains do not start with www.

File: world_universities_and_domains.json [hunk 3]
```diff
@@ -45059,25 +45059,25 @@
...
-      "http://www.koforiduapoly.edu.gh/"
+      "https://www.ktu.edu.gh/"
     ],
-    "name": "Koforidua Polytechnic",
+    "name": "Koforidua Technical University",
     "alpha_two_code": "GH",
     "state-province": null,
     "domains": [
-      "koforiduapoly.edu.gh"
+      "www.ktu.edu.gh"
```
Comment (yigitguler) [inline world_universities_and_domains.json:45068]: emails do not start with www.

File: world_universities_and_domains.json [hunk 4]
```diff
@@ -45131,13 +45131,13 @@
...
-      "http://www.tpoly.edu.gh/"
+      "https://www.ttu.edu.gh/"
     ],
-    "name": "Takoradi Polytechnic",
+    "name": "Takoradi Technical University",
     "alpha_two_code": "GH",
     "state-province": null,
     "domains": [
-      "tpoly.edu.gh"
+      "www.ttu.edu.gh"
```
Comment (yigitguler) [inline world_universities_and_domains.json:45140]: same here.

**Comment by yigitguler** [comment]
Thank you!

**Pull request merged**